### ISSUE TITLE
Meta: Re-evaluate All PKGRECOM/PKGSUG

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-abbs#1918]
Rationale
--------------

The current semantic interpretation of `$PKGRECOM` (Recommends) is something along the line of "soft dependency", which refers to package(s) that could enhance a package's functionalities if installed - and they are not installed automatically in our distribution, as specified in `/etc/apt/apt.conf.d/99synaptic`:

```
APT::Install-Recommends "false";
```

The issue stands that this semantic meaning should be given by `$PKGSUG` (Suggests) instead of "Recommends". Debian specifies in their Policy Manual that:

"This is used to declare that one package may be more useful with one or more others. Using this field tells the packaging system and the user that the listed packages are related to this one and can perhaps enhance its usefulness, but that installing this one without them is perfectly reasonable."

TL;DR: We have misused "Recommends" and "Suggests", time to get them fixed for good.

Plan of Action
--------------------

- [ ] Re-evaluate all `$PKGRECOM` and `$PKGSUG` in the tree.
    - [ ] Make some dependencies of the `-base` packages "Recommends", allowing for more user freedom.
- [ ] Remove `/etc/apt/apt.conf.d/99synaptic`.

Branch and Timeframe
-----------

Explosive -> Testing, Summer 2019.

**Comment by jyhi** [comment]
Supplementary information:

> **Recommends**
>
>This declares a strong, but not absolute, dependency.
>
> The Recommends field should list packages that would be found together with this one in all but unusual installations.
>
> **Suggests:**
>
> This is used to declare that one package may be more useful with one or more others. Using this field tells the packaging system and the user that the listed packages are related to this one and can perhaps enhance its usefulness, but that installing this one without them is perfectly reasonable.

-- https://www.debian.org/doc/debian-policy/ch-relationships.html#binary-dependencies-depends-recommends-suggests-enhances-pre-depends

**Comment by MingcongBai** [comment]
Addressed with a batch job back in May, 2020 (for instance, 109176c42ae1e7d34f3acfad5c0c47c6bf26434d). Closing.
